### ISSUE TITLE
docker compose with Postgresql jar driver

### ISSUE THREAD

**Issue by robomotic** [KnowageLabs/Knowage-Server-Docker#11]
Hello there,
is this the correct approach to load the JAR file?
Is not working for me:

docker-compose.yaml

> knowage:
>     build: knowage/.
>     links:
>         - knowagedb:db
>     ports:
>         - "8080:8080"
>     environment:
>         - WAIT_MYSQL=true
> 
> knowagedb:
>     image: mysql:5.5
>     environment:
>         - MYSQL_USER=knowageuser
>         - MYSQL_PASSWORD=knowagepassword
>         - MYSQL_DATABASE=knowagedb
>         - MYSQL_ROOT_PASSWORD=knowagerootpassword

knowage/Dockerfile:

> FROM knowagelabs/knowage-server-docker:6.1.1
> 
> # https://jdbc.postgresql.org/download.html
> COPY postgresql.jar /home/knowage/apache-tomcat-7.0.57/lib/postgresql.jar

When I then add the data source the Test function with driver:org.postgresql.Driver, I setup a correct URL with username and password but it goes idle and nothing shows up in the logs.

**Comment by aportosa** [comment]
Are you trying to build a custom version of Knowage docker file or just copy the driver while the container is already running?

**Comment by robomotic** [comment]
Yes indeed as you can see I just created a dockerfile which inherits from knowage-server-docker:6.1.1 and before running just adds the jar file. 
I can also try to run the original dockerfile and add the jar file at runtime to see if it makes any difference but very unlikely .....
